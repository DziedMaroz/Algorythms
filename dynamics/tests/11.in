9054
77
054
4

60
28
521
940
53

914


35
5
2

21
33
98


03
42

154
49
000

8
276
5
30

68
00
7
85

2
276


8


9

2
5
82
0
87

385
5
9


125
949


7
742

05
761
379
9

19

2
2
34
85
680
60
