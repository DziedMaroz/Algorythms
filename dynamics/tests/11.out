2
9 054
